### DISCUSSION TITLE
How can i convert the Primeng Table ui to json format to bind it dynamically?

### DISCUSSION THREAD

**Discussion by M-Sai-Charan** [primefaces/community#461] [PrimeNG]
Hi All,

I am using primeng table, to display multiple records on ui, so with static html i created a primeng table, now i want to create a json format of the table records to bind it on html dynamically.

I am attaching the stackblitz url for reference :-

https://stackblitz.com/edit/primeng-tablecolgroup-demo-xnakae?file=src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.component.ts]

Please help me on this issue.

Thanks in advance

**Comment by cetincakiroglu** [comment]
Hi,

What do you mean by "convert the PrimeNG table UI to json"?
